cluster/gce/gci/health-monitor.sh

```diff
@@ -33,22 +33,22 @@ function container_runtime_monitoring {
   # dockershim is still part of kubelet today. When kubelet is down, crictl pods
   # will also fail, and docker will be killed. This is undesirable especially when
   # docker live restore is disabled.
-  local healthcheck_command="docker ps"
+  local healthcheck_command=(docker ps)
```

Comment:
instead of relying on shell splitting behavior we use an explicit array / argv for the healthcheck command

this is the reason for all of the changes related to `healthcheck_command`

Reply:
At some point (soon) we may want to reverse the default behavior from docker to crictl.

Reply:
SGTM

can you approve? :upside_down_face: